src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({
+  a = 0,
+  b = 0,
+}) {
```

Comment:
also I think we should use this for default arguments 
Sum.defaultProps = {
  a: 0,
  b: 0,
}

Reply:
Fixes were added. Thanks!